eksconfig/clusterautoscaler.go

```diff
@@ -39,6 +45,12 @@ func (spec *ClusterAutoscalerSpec) Default(cfg *Config) {
 	if spec.MaxNodes == 0 {
 		spec.MaxNodes = 100
 	}
+	if spec.Resources.Requests.Cpu().IsZero() {
```

Comment:
It's good to have these fields in the spec, but they're not actually used anywhere. Hint: they need to be in the gotemplate.